event_webhook_types.go

```diff
@@ -1048,3 +1064,17 @@ type EventLabel struct {
 	Type        string `json:"type"`
 	GroupID     int    `json:"group_id"`
 }
+
+// StateID identifies the state of an issue or merge request.
+//
+// There are no GitLab API docs on the subject, but the mappings can be found in GitLab's codebase:
+// https://gitlab.com/gitlab-org/gitlab-foss/-/blob/ba5be4989e/app/models/concerns/issuable.rb#L39-42
+type StateID int
+
+const (
+	StateIDNone   StateID = 0
+	StateIDOpen   StateID = 1
+	StateIDClosed StateID = 2
+	StateIDMerged StateID = 3
+	StateIDLocked StateID = 4
+)
```

Comment:
Can you put these (`StateID` and the consts) at the top of the file? Thanks!

Reply:
Done